pkg/config/config.go

```diff
@@ -1364,24 +1364,11 @@ func JitterFrequency(freqInSec time.Duration) time.Duration {
 }
 
 func calculateCollectorURL(licenseKey string, staging bool) string {
-	if staging {
-		return calculateCollectorStagingURL(licenseKey)
+	if license.IsFederalCompliance(licenseKey) {
```

Comment:
I guess it is not possible to use staging  federal licenses on staging, right?

Reply:
Yes, federal licenses are only for PROD env.